C/c4Database.cc

```diff
@@ -377,6 +377,22 @@ bool c4db_purgeDoc(C4Database *database, C4Slice docID, C4Error *outError) {
     return false;
 }
 
+uint64_t c4db_nextDocExpiration(C4Database *database)
+{
+	KeyStore expiryKvs(database, "expiry");
+	DocEnumerator e(expiryKvs);
+	while (e.next()) {
```

Comment:
Number keys are guaranteed to sort before strings, so I don't think you need a `while` loop. Instead just get the first item in the enumeration, and if its key is a number return the numeric value, else give up and return 0.

It looks like you could just change `while` to `if` on this line and get that behavior.